hig.js

```diff
@@ -87,17 +88,41 @@ function speak(phrases) {
 		return false;  	
 	}
 	var p = phrases.shift();
-	var v = 'Alex';
-	if (p.name == ANNOUNCER) {
-		v = 'Samantha';
-	} 
+	if(os == 'mac'){  //mac male voice to talk as hockey player
+		var v = 'Alex';
+			if (p.name == ANNOUNCER ) { //windows female voice to talk as announcer 
+				v = 'Samantha';
+		} 
+	}
+	// else if (os=='win32'){ //windows male voice to talk as hockey player
+		// var v = 'David';
+			// if (p.name == ANNOUNCER ) { //linux female voice to talk as announcer 
+				// v = 'Zira';
+		// }
+	// }
+	// else if (os == 'Darwin'){ // Linux Male voice to talk as hockey player
```

Comment:
I don't understand what is going on here, I think I'm looking at the latest, but there appears to be a bunch of content commented out.

Reply:
Yeah, I am still working on adding voices.  Also, thank you I will fix the OS descriptions.

Reply:
I removed the comments from the win32 condition but am still working on getting it to choose anything besides the default announcer.